src/main/java/Duke.java

```diff
@@ -1,10 +1,346 @@
+
+
+import java.util.ArrayList;
+import java.util.Scanner;
+import duke.task.Todo;
+import duke.task.Task;
+import duke.task.TestTask;
+import duke.task.Event;
+import duke.task.Deadline;
+import duke.exception.DukeException;
+
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class Duke {
+    static int listLength;
+
+    static Task task;
+
+
+    static String userInput;
+    static String command;
+    static String description = "No description is needed for this command";
+    public static final String SEPARATOR = "-----------------------------------";
+    private static ArrayList<Task> tasks;
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
+
         System.out.println("Hello from\n" + logo);
+
+
+
+
+
+        setTasks();
+
+
+        Scanner in = new Scanner(System.in);
+        userInput = in.nextLine();
+
+        while (!userInput.equals("bye")) {
+            try {
+                if (userInput.equals("echo")) {
+
+                    echo(userInput);
+
+                } else if (userInput.length() >= 3 && userInput.substring(0, 3).equals("add")) {  // Add the task into the list
+
+                    addTask(userInput);
+
+                } else if (userInput.equals("list")) {      /* Show the list to user which contains indicators to indicate that whether this task is done or have a deadline or is an event with given time */
+
+
```

Comment:
Occasionally you have two empty lines followed by functional code, but there are also single lines as separators. Refer to the standard [if-else layout](https://se-education.org/guides/conventions/java/basic.html#layout) and remove empty lines that do not serve any purpose.